cli/src/main/java/com/box/l10n/mojito/cli/command/BranchRemoveCommand.java

```diff
@@ -0,0 +1,63 @@
+package com.box.l10n.mojito.cli.command;
+
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.ConsoleWriter;
+import com.box.l10n.mojito.cli.command.param.Param;
+import com.box.l10n.mojito.rest.client.AssetClient;
+import com.box.l10n.mojito.rest.client.RepositoryClient;
+import com.box.l10n.mojito.rest.entity.Branch;
+import com.box.l10n.mojito.rest.entity.Repository;
+import org.fusesource.jansi.Ansi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.util.HashSet;
+import java.util.Set;
+
+@Component
+@Scope("prototype")
+@Parameters(commandNames = {"branch-remove", "br"}, commandDescription = "remove branch")
+public class BranchRemoveCommand extends Command {
+    /**
+     * logger
+     */
+    static Logger logger = LoggerFactory.getLogger(BranchRemoveCommand.class);
+
+    @Autowired
+    ConsoleWriter consoleWriter;
+
+    @Autowired
+    RepositoryClient repositoryClient;
+
+    @Autowired
+    AssetClient assetClient;
+
+    @Autowired
+    CommandHelper commandHelper;
+
+    @Parameter(names = {Param.REPOSITORY_LONG, Param.REPOSITORY_SHORT}, arity = 1, required = true, description = Param.REPOSITORY_DESCRIPTION)
+    String repositoryParam;
+
+    @Parameter(names = {"-b", "--branch"}, arity = 1, required = true, description = "branch")
```

Comment:
would you mind introducing constants (similar to REPOSITORY_SHORT) since the branch param is now used in multiple place?

Reply:
Sure, moving to CONSTANTS